cluster-operator/src/main/java/io/strimzi/operator/cluster/model/TopicOperator.java

```diff
@@ -235,9 +239,26 @@ public Deployment generateDeployment() {
         return varList;
     }
 
+    public static String topicOperatorServiceAccountName(String name) {
```

Comment:
is it better calling the parameter `cluster` as for the other `xxxName` methods. You are using it passing the cluster name.